### DISCUSSION TITLE
Build fails with astro-compress plugin

### DISCUSSION THREAD

**Discussion by jorgeteixe** [arthelokyo/astrowind#301] [Q&A]
Running `npm run dev` returns:

```
 error   EACCES: permission denied, scandir '/opt/jc'
Error: EACCES: permission denied, scandir '/opt/jc'
```

I've seen some comments related to the issue here: https://community.cloudflare.com/t/getting-a-permissions-error-when-deploying-astro-site-to-cf-pages/571528/5

**Comment by jorgeteixe** [comment]
As mentioned on the post, I've tested and pinning it to the previous minor with `"astro-compress": "2.0.14",` on the `package.json` mitigates the issue.

**Comment by prototypa** [comment]
Hello everyone, we know about the long delay. We were working on a complex project, we are back. In the discussion area we will present the new fixed and defined update schedule. Thanks for the patience.

**Comment by jorgeteixe** [comment]
This has been updated with the latest dependencies fix:  commit [0fb6c5](https://github.com/onwidget/astrowind/commit/0fb6c57826d211d019c0605c27a5e6302091887b)

**Reply by jorgeteixe** [reply]
Thank you @prototypa :)
